include/dao/db_manager.js

```
@@ -185,6 +199,91 @@ module.exports = function DBManagerModule(pb) {
                 };
                 cb(err, result);
             });
+        }
+
+
+        /**
+         * Sorts through all created indices and drops any index not declared in pb.config
+         * @method dropUnconfiguredIndices
+         * @param {Array} procedures
+         * @param {Function} cb
+         */
+        this.dropUnconfiguredIndices = function(procedures, cb) {
+            var self = this;
+            //to prevent a cirular dependency we do the require for DAO here.
+            var DAO = require('./dao.js')(pb);
+            this.getStoredIndices(function(err, storedIndices) {
```

The error is not handled